demo/calculator/run.py

```diff
@@ -1,7 +1,8 @@
 import gradio as gr
-
+import time, random
 
 def calculator(num1, operation, num2):
+    time.sleep(5 + random.random() * 3)
```

Comment:
Revert back before merging. You can use `demo/fake_gan` to test this out btw which already includes a short time sleep